libraries/azure-app-configuration-importer/src/appConfigurationImporter.ts

```diff
@@ -43,27 +43,86 @@ export class AppConfigurationImporter {
    * @param timeout - Seconds of entire import progress timeout
    * @param progressCallback - Callback for report the progress of import
    * @param importMode - Determines the behavior when importing key-values. The default value, 'All' will import all key-values in the input file to App Configuration. 'Ignore-Match' will only import settings that have no matching key-value in App Configuration.
-   * @param dryRun - When dry run is enabled, no updates  will be performed to App Configuration. Instead, any updates that would have been performed in a normal run will be printed to the console for review
+   * @returns Promise<void>
    */
   public async Import(
     configSettingsSource: ConfigurationSettingsSource,
     timeout: number,
     strict = false,
     progressCallback?: (progress: ImportProgress) => unknown,
+    importMode?: ImportMode
+  ): Promise<void> {
+    if (importMode === undefined) {
+      importMode = ImportMode.IgnoreMatch;
+    }
+
+    this.validateImportMode(importMode);
```

Comment:
Looks like we will validateImportMode twice in Import function and in function to GetConfigurationChanges